test/michael_scott_queue/stm_michael_scott_queue.ml

```diff
@@ -1,9 +1,8 @@
+open QCheck
```

Comment:
These changes should not be in this PR.   It is good practise to always rebase PRs against the main.  It saves time from reviewers.